NuKeeper/Collaboration/CollaborationFactory.cs

```diff
@@ -74,7 +74,7 @@ public class CollaborationFactory : ICollaborationFactory
 
                 if (reader != null)
                 {
-                    _nuKeeperLogger.Normal($"Collaboration platform specified as '{_platform}'");
+                    _nuKeeperLogger.Normal($"Collaboration platform specified as '{reader.Platform}'");
```

Comment:
Does `_platform` output a wrong value? Why change this?

Reply:
Yes... it was null, and is first set after this method is executed